.github/workflows/tests.yml

```diff
@@ -38,19 +38,11 @@ jobs:
           python-version: ["3.9", "3.10"]
           requirements-level: [pypi]
           db-service: [postgresql13]
-          search-service: [opensearch2,elasticsearch7]
```

Comment:
what is the reason for removing this ?

Reply:
We're not actually using ElasticSearch in the module. TBH we're not even using the DB, this is a very simple module that just sends Celery tasks and emails.